### ISSUE TITLE
Noticed a few moved and dead links

### ISSUE THREAD

**Issue by davidsharp** [avivace/awesome-gbdev#6]
Although I'd be happy to go through and update what I can, when I get the chance.

**Comment by avivace** [comment]
Yep. Some resources are mirror of mirrors of ~20 years old sites. Consistency and availability would be a great add. Maybe we should write a list of problematic/dead links?

**Comment by svendahlstrand** [comment]
Using my script from #10 I found the following dead links:

```
000 http://siobyte.xyz/1-gameboy-tutorial/
404 http://fms.komkon.org/gameboy/Tech/Carts.html
404 http://marc.rawer.de/gameboy/Docs/GBProject.pdf
404 http://www.personal.triticom.com/~erm/gameboy/
```

**Comment by jedahan** [comment]
Would be cool if we could replace those links with archive.org / waybackmachine links if possible

would be cool if we could script adding current links to submit to the waybackmachine for future posterity

**Comment by lmmendes** [comment]
From the 4 links @svendahlstrand mentions only one is broken the siobyte.xyz expired or doesn't exist (bug got fixed with the #11 pull request ). The other 3 links still exist in the web in the following form but seems that they are already fixed in the master README.md

From the 4 links above only one is dead in and the other 3 seem to be working with minor changes.

Old: http://siobyte.xyz/1-gameboy-tutorial/
New: Domain dead or doesn't exist

Old: http://www.personal.triticom.com/~erm/gameboy/ 
New: http://www.personal.triticom.com/~erm/GameBoy/

Old: http://fms.komkon.org/gameboy/Tech/Carts.html 
New: http://fms.komkon.org/GameBoy/Tech/Carts.html

Old: http://marc.rawer.de/gameboy/Docs/GBProject.pdf
New http://marc.rawer.de/Gameboy/Docs/GBProject.pdf

I suppose that this issue can be closed.
